crates/muvm/Cargo.toml

```diff
@@ -15,6 +15,7 @@ byteorder = { version = "1.5.0", default-features = false, features = ["std"] }
 env_logger = { version = "0.11.3", default-features = false, features = ["auto-color", "humantime", "unstable-kv"] }
 krun-sys = { path = "../krun-sys", version = "1.9.1", default-features = false, features = [] }
 log = { version = "0.4.21", default-features = false, features = ["kv"] }
+neli = { version = "0.7.0-rc2" }
```

Comment:
```suggestion
neli = { version = "0.7.0-rc2", default-features = false, features = [] }
```

Reply:
Ah, thanks, I had no idea about the whole "default features" mechanism.